src/main/views/cookies.njk

```diff
@@ -205,6 +219,20 @@
           </td>
           <td class="text-area">{{ t('When you close your browser') }}</td>
         </tr>
+        <tr>
+          <td>X_CMC</td>
```

Comment:
this doesn't exist

Reply:
It was what was in the prototype. What does ARRAffinity do? Ill need a description

Reply:
same description is fine

Reply:
👍